compiler/include/InitNormalize.h

```diff
@@ -165,6 +158,8 @@ class InitNormalize {
   InitPhase       mPhase;
   BlockType       mBlockType;
   BlockType       mPrevBlockType;
+
+  std::set<DefExpr*> implicitFields;
```

Comment:
Not the end of the world, and we don't have a formal convention, but I note that the
other member fields of this particular class use the "m" prefix (for "member") to help
distinguish them from procedure locals within methods.